__tests__/app.test.js

```diff
@@ -109,6 +109,65 @@ describe("/api", () => {
             });
         });
       });
+      describe("PATCH", () => {
+        test("200: sends an object of the updated article if given positive vote number", () => {
+          return request(app)
+            .patch("/api/articles/5/")
+            .send({ inc_votes: 1 })
+            .expect(200)
+            .then(({ body }) => {
+              const { article } = body;
+              expect(article).toEqual({
```

Comment:
consider changing toEqual to a less brittle matcher (imagine if in the future another property is added to each article, you don't want existing tests to fail). toMatchObject or objectContaining are good options here